#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_add_system_identifiers_to_catalog.dpatch by Adam Di Carlo <aph@debian.org>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.{2,3}/catalog.xml: Adds system identifiers to the catalog.
## DP: * docbook-4.x/docbook.cat: Likewise.
## DP:
## DP: <URL:http://bugs.debian.org/301157>
## DP: <URL:http://bugs.debian.org/304057>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-8>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.4-1>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-2>

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.1.2/docbook.cat docbook-xml-4.5/docbook-4.1.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.1.2/docbook.cat	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.1.2/docbook.cat	2007-04-18 20:37:36.000000000 +0200
@@ -23,6 +23,8 @@
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.1.2/docbookx.dtd" "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.1.2//EN" "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.2/catalog.xml docbook-xml-4.5/docbook-4.2/catalog.xml
--- docbook-xml-4.5~/docbook-4.2/catalog.xml	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/catalog.xml	2007-04-18 20:35:46.000000000 +0200
@@ -28,6 +28,12 @@
 <public publicId="-//OASIS//DTD DocBook XML V4.2//EN"
         uri="docbookx.dtd"/>
 
+<system systemId="http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
 <!-- ...................................................................... -->
 <!-- DocBook modules ...................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.2/docbook.cat docbook-xml-4.5/docbook-4.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.2/docbook.cat	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/docbook.cat	2007-04-18 20:38:16.000000000 +0200
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.2/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.2//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
diff -urNad docbook-xml-4.5~/docbook-4.3/catalog.xml docbook-xml-4.5/docbook-4.3/catalog.xml
--- docbook-xml-4.5~/docbook-4.3/catalog.xml	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/catalog.xml	2007-04-18 20:35:46.000000000 +0200
@@ -28,6 +28,12 @@
 <public publicId="-//OASIS//DTD DocBook XML V4.3//EN"
         uri="docbookx.dtd"/>
 
+<system systemId="http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.3/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
 <!-- ...................................................................... -->
 <!-- DocBook modules ...................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.3/docbook.cat docbook-xml-4.5/docbook-4.3/docbook.cat
--- docbook-xml-4.5~/docbook-4.3/docbook.cat	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/docbook.cat	2007-04-18 20:38:47.000000000 +0200
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.3/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.3//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
diff -urNad docbook-xml-4.5~/docbook-4.4/docbook.cat docbook-xml-4.5/docbook-4.4/docbook.cat
--- docbook-xml-4.5~/docbook-4.4/docbook.cat	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.4/docbook.cat	2007-04-18 20:39:13.000000000 +0200
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.4/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.4//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
diff -urNad docbook-xml-4.5~/docbook-4.5/docbook.cat docbook-xml-4.5/docbook-4.5/docbook.cat
--- docbook-xml-4.5~/docbook-4.5/docbook.cat	2007-04-18 20:35:45.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/docbook.cat	2007-04-18 20:39:48.000000000 +0200
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.5/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.5//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
